server/controllers/caseNotesController.ts

```diff
@@ -117,15 +113,9 @@ export default class CaseNotesController {
   public displayAddCaseNote(): RequestHandler {
     return async (req: Request, res: Response, next: NextFunction) => {
       const userToken = res.locals.user.token
-      const { firstName, lastName, prisonerNumber, prisonId, restrictedPatient, cellLocation } =
-        req.middleware.prisonerData
+      const { firstName, lastName, prisonerNumber, prisonId, cellLocation } = req.middleware.prisonerData
       const prisonerBannerName = formatName(firstName, null, lastName, { style: NameFormatStyle.lastCommaFirst })
 
-      // If user cannot view this prisoner's case notes, redirect to 404 page
-      if (!canViewCaseNotes(res.locals.user, { prisonId, restrictedPatient })) {
-        return next()
-      }
-
```

Comment:
Again removing this in favour of the `checkPrisonerInCaseload` (allowGlobal: false) middleware